src/components/QuestionComponent/styled.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+import {pxToRem} from '../../utils/unit';
+
+const StyledQuestionComponent = styled.section`
+	width: 25rem;
+	height: 5rem;
+	margin-bottom: ${pxToRem(8)};
+	padding: 0.5em 1em;
+	border: 1px solid rgba(0, 0, 0, 0.3);
+	border-radius: ${pxToRem(8)};
+	background: #517664;
+	color: #fff;
+	font-size: 1em;
```

Comment:
If you work with `em` instead of `rem` you should know that `em` could result in some interesting styling issues that later you may not want to have. [See this article for further for examples.](https://www.digitalocean.com/community/tutorials/css-rem-vs-em-units) Just keep this in mind. :)

Reply:
Thanks for your comment!
This is on purpose :)